source/upgrade.txt

```diff
@@ -0,0 +1,156 @@
+.. _java-upgrade-driver:
+
+=======================
+Upgrade Driver Versions
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+In this section, you can identify the changes you need to make to your
+application to upgrade your driver to a new version or to transition from
+the legacy API to the current API.
+
+When you upgrade your driver version or use multiple versions of the
+driver, if there are any breaking changes between the two versions that you
+have not addressed in the :ref:`Breaking Changes <java-breaking-changes>`
```

Comment:
Sometime in the past couple months, I discovered that this is not a reliable substitution mechanism (sometimes the parser cannot locate the title and therefore omits it), so I've avoided using it since then.